qemu/tests/cfg/nic_hotplug.cfg

```diff
@@ -70,8 +70,15 @@
             requires_root = yes
             type = migration_after_nichotplug
             kill_vm = yes
-            nics = ""
```

Comment:
Hold on, for 'after_nichotplug', we still need this line, otherwise `nics = nic1`
Other LGTM.